components/dropdown/src/auro-dropdown.js

```diff
@@ -229,14 +237,19 @@ export class AuroDropdown extends LitElement {
 
   firstUpdated() {
     this.floater.configure(this);
+    this.bibContent = this.floater.element.bib;
+    if (this.mobileFullscreenBreakpoint) {
+      const docStyle = getComputedStyle(document.documentElement);
+      const breakpointValue = docStyle.getPropertyValue(DESIGN_TOKEN_BREAKPOINT_PREFIX + this.mobileFullscreenBreakpoint);
```

Comment:
I think we need to make sure that `this.mobileFullScreenBreakpoint` is a valid value before we run this code. Otherwise, we will try to do getPropertyValue on something invalid and pass that down to the the bib.

Reply:
Additional:

I think auro-dropdown should just pass the breakpoint string (sm, md, etc..) down to auro-dropdownbib and all the functional code can live there.

Additionally, rather than doing this on firstUpdated this should run inside:

```
updated(changedProperties) {
  if (changedProperties.has('mobileFullscreenBreakpoint') {
    // execute the code here...
  }
}
```

That way we handle the scenario where a consumer might hypothetically change the breakpoint to trigger mobile after the component get's loaded into the DOM.